index.html

```diff
@@ -46,39 +46,47 @@
 
     <main>
         <div class="homepage-category">
-            <a href="">
+            <a href="./pages/product-listing/product-listing.html">
                 <div class="category-item">
                     <img src="./assets/gr2.png" alt="">
                     <div class="textoverlay-container">
                         <p class="overlay-text">Ring</p>
                     </div>
                 </div>
             </a>
-            <a href="">
-                <div class="category-item">
-                    <img src="./assets/gr2.png" alt="">
-                </div>
-            </a>
-            <a href="">
+            <a href="./pages/product-listing/product-listing.html">
                 <div class="category-item">
                     <img src="./assets/gr2.png" alt="">
```

Comment:
Don't leave Alt = "" blank, always fill it with some text, so we will get to know which image is missing if image doesn't loads.